### PR TITLE
Support raw json fields

### PR DESCRIPTION
**Pull request by kwapik**
This is a port of @cameronbrunner fix to the original repo. See https://github.com/shurcooL/graphql/pull/41
It also contains fixes to `quest_test.go`

### CODE REVIEW

**Comment by kwapik** [comment]
@hgiasac PTAL, I'd be very grateful to have this accepted.

File: internal/jsonutil/graphql.go [hunk 1]
```diff
@@ -97,24 +104,36 @@ func (d *decoder) decode() error {
...
+				tok, err = d.tokenizer.Token()
+				if err == io.EOF {
+					return errors.New("unexpected end of JSON input")
+				} else if err != nil {
+					return err
+				}
 			}
 
-		// Are we inside an array and seeing next value (rather than end of array)?
+			// Are we inside an array and seeing next value (rather than end of array)?
```
Comment (bclermont) [inline internal/jsonutil/graphql.go:136]: look like you broke that indentation

File: query_test.go [hunk 2]
```diff
@@ -263,7 +263,7 @@ func TestConstructMutation(t *testing.T) {
 					Content:   ReactionContentThumbsUp,
 				},
 			},
-			want: `mutation($input:AddReactionInput!){addReaction(input:$input){subject{reactionGroups{users{totalCount}}}}}`,
+			want: `mutation ($input:AddReactionInput!){addReaction(input:$input){subject{reactionGroups{users{totalCount}}}}}`,
```
Comment (bclermont) [inline query_test.go:266]: you added a space here

**Comment by hgiasac** [comment]
LGTM. Let's solve @bclermont comments, then we can merge this

**Comment by kwapik** [comment]
@hgiasac @bclermont sorry, I missed your review. Will fix this today or tomorrow.

**Comment by kwapik** [comment]
Made changes. Unfortunately I can't re-request review. @bclermont @hgiasac

**Comment by kwapik** [comment]
Also needed to adjust error message to meet #8 requirements.

**Review by hgiasac — APPROVED** [review]
Thank you!

**Pull request merged**